k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -59,8 +41,15 @@ spec:
           value: 3CjmRhu718gT1Wkba3LfdqX5pfYuBdaMPLd7ENUga5dm
```

Comment:
It's probably not a good idea to include a valid default keypair, because someone might use it. Maybe just put something like `<insert a valid public key here>` and `<insert a valid private key corresponding to the above public key>`.